docs/dev_database/common/ppips.rst

```diff
@@ -0,0 +1,61 @@
+===========
+ppips files
+===========
+
+The *ppips files* are generated for every FPGA :term:`tile <tile>` type.
```

Comment:
Need a description of why these files are important.  They inform the tool that when emit a PIP that is missing from segbits_X, but present in ppips_X to not generate and error, and when going in reverse allow the tools to know this edge is always on.

Reply:
I've added appropriate information